core/src/main/java/hudson/lifecycle/ExitLifecycle.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License
```

Comment:
Not valid without copyright holder IIRC. IANAL though

Reply:
ok, added self.